apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -124,6 +133,15 @@ export class ApplicationService {
       await Promise.all(promises)
     }
 
+    await this.sendEmail(
+      {
+        name: user.name,
+        address: application.email,
+      },
+      appModel.id,
+      `Umsókn þín er móttekin og er nú í vinnslu.`,
```

Comment:
Maybe not a part of this PR, but I think that it would be super nice if the next step in emails is adding a link to status page.